src/vmm/src/logger/logging.rs

```diff
@@ -319,6 +320,7 @@ mod tests {
             ("warning", LevelFilter::Warn),
             ("error", LevelFilter::Error),
         ] {
+            let (level, level_enum) = pair;
```

Comment:
Cannot the destructuring be done above? pair does not look used after.

Also these changes should go in the previous commit?

Reply:
the 3rd commit looked like an optimization of a working functionality so created it as a separate commit but I can merge the 2 commits if they don't look right?

Reply:
merged the unit test commits.